File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.py
import sys
input = sys.stdin.readline

C = int(input())

for _ in range(C):
    N, *arr = map(int, input().split())

    # 평균 구하기
    avg = sum(arr) / N

    # 평균을 넘는 학생 수
    cnt = 0
    for i in range(N):
        if arr[i] > avg:
            cnt += 1

    # 비율
    print(f'{cnt/N*100:.3f}%')

File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.js
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().split('\n');

const c = Number(input[0]);

for (let i = 1; i < c + 1; i++) {
  const [n, ...score] = input[i].split(' ').map(Number);

  // 평균
  const sumScore = score.reduce((acc, cur) => acc + cur, 0);
  const avg = sumScore / n;

  // 평균이 넘는 학생
  let students = 0;
  for (let j = 0; j < n; j++) {
    if (score[j] > avg) {
      students++;
    }
  }

  console.log(`${((students / n) * 100).toFixed(3)}%`);
}
